### PR TITLE
tests/api tweaks

### PR DESCRIPTION
**Pull request by iamigo**

### CODE REVIEW

File: tests/api/v1/samples/sampleCount.js [hunk 1]
```diff
@@ -121,23 +120,23 @@ describe('Sample Count:', () => {
...
-        .end((delErr  , res ) => {
+        .end((delErr, res) => {
           if (delErr) {
             return done(delErr);
           }
 
           api.get(`${path}/subjects/${tu.namePrefix}Subject`)
           .set('Authorization', token)
-          // .expect(constants.httpStatus.OK)
+          /* .expect(constants.httpStatus.OK) */
```
Comment (jgraff2) [inline tests/api/v1/samples/sampleCount.js]: should this be commented?

Comment (iamigo) [inline tests/api/v1/samples/sampleCount.js]: idk - i was just changing the formatting to avoid the style error here

File: tests/api/v1/subjects/deleteHierarchy.js [hunk 2]
```diff
@@ -20,13 +19,12 @@ const Subject = tu.db.Subject;
...
 
 describe(`api: DELETE ${path}`, () => {
-  let token;
-
   const par = { name: `${tu.namePrefix}NorthAmerica`, isPublished: true };
   const chi = { name: `${tu.namePrefix}Canada`, isPublished: true };
   const grn = { name: `${tu.namePrefix}Quebec`, isPublished: true };
   let ipar = 0;
   let ichi = 0;
+  let token;
```
Comment (jgraff2) [inline tests/api/v1/subjects/deleteHierarchy.js]: why was this added?

Comment (iamigo) [inline tests/api/v1/subjects/deleteHierarchy.js]: was just moved a few lines down so that constants are declared above variables
